tests/modules/nf-core/spring/decompress/test.yml

```diff
@@ -0,0 +1,23 @@
+- name: spring decompress test_spring_decompress_single_end
+  command: nextflow run ./tests/modules/nf-core/spring/decompress -entry test_spring_decompress_single_end -c ./tests/config/nextflow.config
+  tags:
+    - spring
+    - spring/decompress
+  files:
+    - path: output/spring/test.fastq.gz
+      md5sum: e71acb18cb172037d021daf2801e6d18
+    - path: output/spring/test.spring
+    - path: output/spring/versions.yml
```

Comment:
odd that this file would not have a stable MD5 sum, can you confirm?

Reply:
The `nf-core tools modules create-test-yml` command by default does not include the md5sum for the versions.yml file. This means that bumping the version of the tool doesn't require updating the md5sums unless something has changed in the output.

Reply:
seems reasonable, going to resolve comment then